cockroachcloud/cloud-org-sso.md

```diff
@@ -30,31 +30,50 @@ Cloud Organization SSO allows you to customize your SSO configuration to meet yo
 
 - Members sign in using a custom URL that allows only the authentication methods that you have configured.
 - Members can sign in using any enabled authentication method, to help reduce the impact of an IdP outage. If a member signs in using a new method for the first time, they are prompted to optionally update their default method. **This is possible only as long as the members are using the same email address to sign in through each method**.
-- You can [enable multiple authentication methods](configure-cloud-org-sso.html#enable-or-disable-an-authentication-method) simultaneously. You can even add custom authentication methods that connect to IdPs such as Okta or ActiveDirectory through the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
+- You can [enable multiple authentication methods](configure-cloud-org-sso.html#enable-or-disable-an-authentication-method) simultaneously. You can even add custom authentication methods that connect to IdPs such as Okta or ActiveDirectory through the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language), [System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644), and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
 - You can disable any authentication method. To enforce a requirement to use SSO, you can enable only SSO authentication methods and disable password authentication. If you disable password authentication, passwords are not retained.
-- [Autoprovisioning](#autoprovisioning) optionally removes the need to invite members to your organization. Autoprovisioning is disabled by default for each SSO authentication method.
 - You can [restrict the email domains](configure-cloud-org-sso.html#allowed-email-domains) that are allowed to sign in using an SSO authentication method. By default, any email domain is allowed.
+- [SSO Autoprovisioning](#sso-autoprovisioning) can be enabled per SSO authentication method, and automatically creates a {{ site.data.products.db }} organization account when a member successfully authenticates using an SSO authentication method for the first time, with no invitation required.
+- [SCIM Autoprovisioning](#scim-autoprovisioning) automatically creates a {{ site.data.products.db }} organization account when a user is assigned to the SCIM application in your IdP that is connected to your {{ site.data.products.db }} organization.
```

Comment:
Instead of saying SCIM auto-provisioning I think it's preferable to refer to it as SCIM provisioning (and SSO auto-provisioning should be renamed back to auto-provisioning). The reason why this distinction is important is because the automation aspect pertains to automated identity management as a whole (including deprovisioning) and so it's much more than just auto-provisioning (automatically creating identities) which is the case for enterprise login methods.

i.e.
Auto-provisioning
* automated user provisioning 
* may be enabled per SSO auth method 
* streamlines giving users access by automatically creating their identities when they select an SSO auth method

SCIM provisioning 
* automated user management (including automated user provisioning, de-provisioning, group assignment)
* not related to SSO (the user exists but the identity attached to an enterprise login method isn't created until they login for the first time)

Just want to also clarify I think SCIM provisioning should be taken out here and put in a separate doc, a doc that will link to this doc when needed

Reply:
Done